tests/harnesses/ld-floxlib/print-gif-info.c

```diff
@@ -0,0 +1,36 @@
+#include <gif_lib.h>
```

Comment:
where is giflib coming from 🤔

Reply:
https://github.com/flox/flox/pull/733/files#diff-5fad6a349543935cb9b646a5b41d16651bb69232277b614a671b1d611f9d7263R62

Reply:
maybe worth a comment...

Reply:
We stopped using `libgif.so` as we moved to instead linking against `libnixmain.so`, so in one respect this is outdated, but yes the test program is using include files and libraries provided by the packages installed to the environment. That's what we're testing here, so kind of obvious? I guess I don't understand what the comment would say except this is a program to test that we can use a package installed to the env? That's kind of the entire point of the test, so kind of circular/redundant to say so?

Anyway, we have a similar opportunity to comment the `print-nix-version.cc` file ... let me know if you think that's really necessary.

Reply:
I think it would be good to have a comment in the file describing its purpose, esp. when its _the include_ that is the central concern of this binary ...